{{ cookiecutter.project_shortname }}/setup.cfg

```diff
@@ -17,15 +17,22 @@ classifiers =
 [options]
 include_package_data = True
 packages = find:
+python_requires = >=3.7
 zip_safe = False
 install_requires =
     invenio-base>=1.2.5
     invenio-i18n>=1.3.1
 
 [options.extras_require]
 tests =
-    pytest-invenio>=1.4.2
-    sphinx>=4.2.0
+    pytest-invenio>=2.1.0,<3.0.0
+    pytest-black>=0.3.0
```

Comment:
I opened https://github.com/inveniosoftware/pytest-invenio/issues/92 to see if we could integrate in `pytest-invenio` and skip it.